### PR TITLE
Feed substrate oracle.

### PR DESCRIPTION
**Pull request by shaunxw**

### CODE REVIEW

File: src/priceFeeder/substrate/feeder.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import Keyring from '@polkadot/keyring';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { stringToU8a } from '@polkadot/util';
+import BigNumber from 'bn.js';
+
+import logger from '../../logger';
+import { FeederKind, Listing } from '../types';
+
+const createProvider = (endpoint: string): WsProvider | HttpProvider => {
+  if (endpoint.startsWith('wss') || endpoint.startsWith('ws')) {
```
Comment (xlc) [inline src/priceFeeder/substrate/feeder.ts]: `endpoint.startsWith('ws')` also covers `wss` case, same for http & https
if you want to be explicit, can just add a comment

File: src/priceFeeder/acala/currencyIds.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "ACAUSD": "0",
+    "USDUSD": "1",
+	"DOTUSD": "2",
```
Comment (xlc) [inline src/priceFeeder/acala/currencyIds.json]: indent

File: src/priceFeeder/substrate/feeder.ts [hunk 3]
```diff
@@ -3,29 +3,33 @@ import { WsProvider, HttpProvider } from '@polkadot/rpc-provider';
 import Keyring from '@polkadot/keyring';
 import { KeyringPair } from '@polkadot/keyring/types';
 import { stringToU8a } from '@polkadot/util';
-import BigNumber from 'bn.js';
+import BN from 'bignumber.js';
```
Comment (xlc) [inline src/priceFeeder/substrate/feeder.ts:6]: why use bignumber.js over BN.js? I think polkadot.js is using BN.js?

Comment (shaunxw) [reply src/priceFeeder/substrate/feeder.ts:6]: bn.js only deals with integers, and it's a natural choice for libs with only integers like polkadot.js and web3.js.

But for oracle server, it's more receiving floating point price string from api and convert it to int, with accuracy info. bn.js doesn't handle this scenario.

**Review by xlc — APPROVED** [review]

**Pull request merged**